### PR TITLE
feat(19709): Add tablular visualisation of policies, scripts and schemas

### PR DESCRIPTION
**Pull request by vanch3d**
See https://hivemq.kanbanize.com/ctrl_board/57/cards/19709/details/

The PR refactors the main `Data Hub` landing page to offer access to a table-based visualisation of all the policies and their resources, scripts and schema, present in the Edge.

The lists are displayed in separate tabs on the UI and are paginated, as with previous table usages. Filtering is not yet supported.

The policy table offers a principal CTA to create new ones. Similar features are not available for the scripts and schemas (their creation should be handled as part of the policy creation by the Designer). 

The actions on each line of the tables offer deleting the items. An edit button is offered for the policy but not for the resources (again, editing resources is assumed to be handled by the designer) 

### Out-of-scope
- safeguards and feedback on deletion will be done in a subsequent ticket, see https://hivemq.kanbanize.com/ctrl_board/57/cards/19819/details/

### Before
screenshot-localhost_3000-2024 02 28-10_54_58

### After
screenshot-localhost_3000-2024 02 28-10_55_29

screenshot-localhost_3000-2024 02 28-10_55_44

screenshot-localhost_3000-2024 02 28-10_56_03

### CODE REVIEW

**Comment by sfrehse** [comment]
How are these items currently ordered?

File: hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx [hunk 1]
```diff
@@ -100,20 +101,23 @@ const PolicyTable: FC = () => {
...
-              <IconButton
-                size="sm"
-                onClick={() => navigate(`/datahub/${info.row.original.type}/id`)}
-                aria-label={t('policyTable.action.edit')}
-                icon={<FaEdit />}
+              <DataHubListAction
+                onDelete={() =>
+                  deletePolicy
+                    .mutateAsync(info.row.original.id)
+                    .then((e) => console.log('XXXXXX', e))
```
Comment (sfrehse) [inline hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx:117]: Looks like left over.

**Comment by vanch3d** [comment]
As with all the tables, items are not initially sorted in any way. They are displayed as they are produced by the backend. Then all the actionable fields (i.e. all except "Actions") can be used to sort the columns. Don't forget to try the SHIFT+click to do multiple sort conditions.

File: hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx (hunk 1 above)
Comment (vanch3d) [reply hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx:117]: See https://hivemq.kanbanize.com/ctrl_board/57/cards/19819/details/

Comment (sfrehse) [reply hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx:117]: Make sense. I will put some further details into the ticket.

**Review by sfrehse — APPROVED** [review]

**Comment by ryanbateman** [comment]
A question about the text ``"Your transformation code will be listed there"`` in the UI (and equivalents in other screens) - I think (if I understand what this is trying to say) that this writing could be phrased to sound a little more natural. It reads (quite understandably) like non-native English. What's the current process for defining and feedback for this kind of UX writing?

**Pull request merged**